src/lib/optimizer/strategy/predicate_reordering_rule.cpp

```diff
@@ -25,8 +25,10 @@ using namespace opossum;  // NOLINT
 bool is_predicate_style_node(const std::shared_ptr<AbstractLQPNode>& node) {
   if (node->type == LQPNodeType::Predicate) return true;
 
-  // Validate can be seen as a Predicate on the MVCC column
+    // Validate can be seen as a Predicate on the MVCC column
+#if 0
   if (node->type == LQPNodeType::Validate) return true;
+#endif
```

Comment:
Current benchmarking results on vm-nordes. with-reordering includes this early return, without-reordering has it commented out.

![image](https://user-images.githubusercontent.com/13838962/77848278-ee5f3600-71c3-11ea-9fd5-6273a5ee579e.png)

So I guess we will leave the reordering as it is in master? @Bensk1

Reply:
My guess is that the performance issues of Validate that led to this being considered have been resolved in the meantime. As such, even if I'm not asked: yes.